README.md

```diff
@@ -6,6 +6,10 @@ Continuous integration server launches Qserv build and also multi-node integrati
 
 [![Build Status](https://travis-ci.org/lsst/qserv.svg?branch=master)](https://travis-ci.org/lsst/qserv)
 
+[![Code Climate](https://codeclimate.com/github/lsst/qserv/badges/gpa.svg)](https://codeclimate.com/github/lsst/qserv)
+
+[![Issue Count](https://codeclimate.com/github/lsst/qserv/badges/issue_count.svg)](https://codeclimate.com/github/lsst/qserv)
```

Comment:
Is this a LSST-approved tool? I'm not very comfortable with links to random sites appearing in project README.